src/benchJsonOpt.c

```diff
@@ -303,7 +303,20 @@ static int getMetaFromInsertJsonFile(cJSON *json) {
     g_arguments->dbCount = dbSize;
     for (int i = 0; i < dbSize; ++i) {
         SDataBase *database = &(g_arguments->db[i]);
-        cJSON *    dbinfos = cJSON_GetArrayItem(dbs, i);
+        database->dbCfg.minRows = -1;
+        database->dbCfg.maxRows = -1;
+        database->dbCfg.comp = -1;
+        database->dbCfg.walLevel = -1;
+        database->dbCfg.cacheLast = -1;
+        database->dbCfg.fsync = -1;
+        database->dbCfg.replica = -1;
```

Comment:
then if the user did not specify replica in JSON file, we will pick the default value -1 to create a database?

Reply:
we only set the database parameter when its value is larger than or equal to zero, set -1 is default and do nothing when create database